src/components/photoGallery/PhotoGallery.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useEffect } from "react";
+import "./PhotoGallery.scss"
```

Comment:
Semicolon after import:
```suggestion
import "./PhotoGallery.scss";
```